src/app/filecoin-plus/allocators/data/allocatorTable.tsx

```diff
@@ -0,0 +1,59 @@
+import { ArrowUpRight } from '@phosphor-icons/react/dist/ssr'
+import { createColumnHelper } from '@tanstack/react-table'
+
+import { Icon } from '@/components/Icon'
+
+import type { AllocatorProps } from '../components/AllocatorsList'
+
+const columnHelper = createColumnHelper<AllocatorProps>()
+
+export function mapAllocatorsToTableData() {
+  return [
+    columnHelper.accessor('name', {
+      header: 'Organization Name',
+      cell: (info) => info.getValue(),
+    }),
+    columnHelper.accessor('type', {
+      header: 'Type',
+      cell: (info) => info.getValue(),
+    }),
+    columnHelper.accessor('region', {
+      header: 'Region',
+      cell: (info) => info.getValue(),
+    }),
+    columnHelper.accessor('required_replicas', {
+      header: 'Required Replicas',
+      cell: (info) => info.getValue(),
+    }),
+    columnHelper.accessor('required_sps', {
+      header: 'Required SPs',
+      cell: (info) => info.getValue(),
+    }),
+    columnHelper.accessor('apply', {
+      header: 'Apply',
+      meta: {
+        bodyCellStyle: 'text-center text-brand-300',
+      },
+      cell: (info) => {
+        const link = info.getValue()
+        const name = info.row.original.name
+
+        if (!link) {
+          return null
+        }
+
+        return (
+          <a
+            href={link}
+            rel="noopener noreferrer"
+            className="inline-flex items-center justify-center gap-2 focus:brand-outline"
+            aria-label={`Apply for ${name} allocator`}
+          >
+            <span>Apply</span>
+            <Icon component={ArrowUpRight} size={20} />
+          </a>
```

Comment:
I can't test this at the moment since the table isn't showing - But could we use `CustomLink`?